fd/example_test.go

```diff
@@ -44,3 +45,24 @@ func ExampleDerivative() {
 	// f'(0) ≈ 0.9999998333333416
 	// f''(0) ≈ -2.999999981767587
 }
+
+func ExampleJacobian() {
+	f := func(y, x []float64) {
```

Comment:
maybe f := func(output, input)? The current code is clear for you and me, but for those who aren't aware of the convention